packages/angular/projects/ui-angular/src/lib/components/authenticator/components/sign-in/sign-in.component.ts

```diff
@@ -23,10 +18,8 @@ export class SignInComponent {
 
   // translated phrases
   // Support backwards compatibility for legacy key with trailing space
```

Comment:
Are we continuing to support backwards compatibility for legacy key with trailing space?

Reply:
Nope, let me remove that comment!